### PR TITLE
Add Kay And Dragonwhale

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Everything has been tested and works

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -533,6 +562,33 @@ let BattleMovedex = {
...
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, 'Psywave', target);
+			this.add('-anim', source, 'Octazooka', target);
+		},
+		self: {
+			boosts: {
+				def: -1,
+				spd: -1,
```
Comment (CheeseMuffin) [inline mods/seasonal/moves.js:584]: It's supposed to lower speed one stage as well

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -533,6 +562,33 @@ let BattleMovedex = {
...
+		desc: "Lowers the user's Defense and Special Defense by 1 stage.",
+		shortDesc: "Lowers the user's Defense and Sp. Def by 1.",
+		id: "inked",
+		name: "Inked",
+		pp: 5,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, 'Psywave', target);
```
Comment (CheeseMuffin) [inline mods/seasonal/moves.js]: It's just supposed to be Octazooka, Psywave was crossed out on smogon but formatting got messed up when I c/ped it

File: mods/seasonal/moves.js (hunk 1 above)
Comment (ghost) [reply mods/seasonal/moves.js:584]: Done

**Pull request merged**